### PR TITLE
Cut release 5.1.0 #no-public-changes

### PR DESCRIPTION
**Pull request by SD10**
This commit cuts the release for IBAnimatable 5.1.0

### CODE REVIEW

**Comment by SD10** [comment]
@JakeLin It looks like I will need an approval to merge this.

I also did 2 other things in this PR:
- I updated the installation instructions to have a more friendly syntax. This includes updating the `Package.swift` instructions to the latest syntax for SPM.
- I moved the installation instructions higher in the README. I believe it was way too far down for people to find easily.

File: README.md [hunk 1]
```diff
@@ -31,6 +31,42 @@ With `IBAnimatable`, we can design a UI in Interface Builder like what we can do
 
 As a designer, we love Sketch, which is a simple but yet super powerful tool to create UI. However, Sketch can't design interaction, navigation, transition and animation, and we may need another tool like Framer to design some of them. Moreover, to make an App Store ready App, we need to use Xcode and Interface Builder to implement the UI and animations. To speed up the process and minimize the waste, we create `IBAnimatable` to make Interface Builder designable and animatable.   
 
+## How to install
```
Comment (JakeLin) [inline README.md]: I think that's a great idea to move it up.

File: README.md [hunk 2]
```diff
@@ -31,6 +31,42 @@ With `IBAnimatable`, we can design a UI in Interface Builder like what we can do
...
+
+As @DanielAsher mentioned
+> I use carthage update --use-submodules --no-build --no-use-binaries and manually add the both the framework project and the framework as an embedded dependency.
+>This method is robust, and fine-grained, but perhaps not as easy as dragging the built framework into your project.
+
+### Git submodule
+
+Add this repo as a submodule, and add the project file to your workspace. You can then link against `IBAnimatable.framework` for your application target. 
+
 ## Version 5
```
Comment (JakeLin) [inline README.md]: We may add a section called `## Version 5.1`
```
IBAnimatable 5.1 is the latest major release of IBAnimatable. This version supports Swift 4.1. 
```

**Review by JakeLin — APPROVED** [review]
Well done. Happy release 🎉

File: README.md (hunk 2 above)
Comment (SD10) [reply README.md]: Done 👍

**Comment by JakeLin** [comment]
@SD10 thanks, happy releasing 🎉

**Pull request merged**